src/parser/validation.ts

```diff
@@ -0,0 +1,75 @@
+import { RowValidateCallback } from "@fast-csv/parse";
+import { utils } from "ethers";
+
+import { AssetTransfer, CollectibleTransfer, Transfer, UnknownTransfer } from "./csvParser";
+
+export const validateRow = (row: Transfer | UnknownTransfer, callback: RowValidateCallback) => {
+  switch (row.token_type) {
+    case "erc20":
+    case "native":
+      validateAssetRow(row, callback);
+      break;
+    case "erc1155":
+    case "erc721":
+      validateCollectibleRow(row, callback);
```

Comment:
I am new to TS switches, but I'm guessing this means both cases are validated and not that one is skipped.